roles/consul/defaults/main.yml

```diff
@@ -5,7 +5,8 @@ consul_dc: dc1
 consul_servers_group: consul_servers
 consul_advertise: "{{ ansible_default_ipv4.address }}"
 consul_bind_addr: "{{ ansible_default_ipv4.address }}"
-consul_bootstrap_expect: "{{ groups[consul_servers_group] | length }}"
+consul_retry_join: "{% for host in groups[consul_servers_group] | intersect(groups[consul_dc]) %}\"{{ hostvars[host].ansible_default_ipv4.address }}\"{% if not loop.last %}, {% endif %}{% endfor %}"
```

Comment:
fwiw, I've seen consul consistently fail to cluster on `bootstrap/apollo-launch.sh` and every consul agent ends up with a cluster of one member, itself.  We worked around it by defining `consul_join` but this is more flexible.